api/paths/paths.go

```diff
@@ -203,7 +262,7 @@ func (k fileKey) perms() os.FileMode {
 func (k fileKey) key() string {
 	switch k {
 	case Home:
-		return "home"
+		return "hom"
```

Comment:
Hi @codenrhoden,

Yes, if you read the tokens you'll note they're all three characters long. I wanted to fix the `home` token to keep it consistent.